bsd-user/elfload.c

```diff
@@ -777,6 +777,8 @@ int load_elf_binary(struct bsd_binprm *bprm, struct target_pt_regs *regs,
     setup_arg_pages(bprm, info, &bprm->p, &bprm->stringp);
     info->start_stack = bprm->p;
 
+    info->elf_flags = elf_ex.e_flags;
```

Comment:
Sorry to pick a fine nit: but if you can split the 'saving of the elf flags' apart from the 'interpreting them in target_arch_thread.h' that will make upstreaming mildly easier. I plan on upstreaming the 'core + i386' first, and then add architectures and a split like I'm asking for would be a tiny bit easier that way....

Reply:
done.